lib/core/src/lib/viewer/components/viewer.component.spec.ts

```diff
@@ -115,6 +115,23 @@ describe('ViewerComponent', () => {
 
     });
 
+    describe('originalMimeType', () => {
+
+        it('should set alt attribute to originalMimeType when originalMimeType is provided', () => {
+            component.originalMimeType = 'image/png';
+            fixture.detectChanges();
+            const altAttribute = fixture.nativeElement.querySelector('.adf-viewer__mimeicon').getAttribute('alt');
```

Comment:
altAttribute has any type, please set type: 
<img width="928" alt="Screenshot 2023-11-21 at 09 05 27" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/70acc61c-935b-4f8f-90f5-2c60e3bccaf5">

Reply:
The type has been set.